srv/components/ha/corosync-pacemaker/config/cluster_ip.sls

```diff
@@ -40,6 +40,11 @@ Setup ClusterIP resouce:
   cmd.run:
     - name: pcs resource create ClusterIP ocf:heartbeat:IPaddr2 ip={{ pillar['cluster']['cluster_ip'] }} mac=$(echo "01:00:5e:`echo {{ grains['hwaddr_interfaces'][data_if] }} | cut -d ":" -f 1-3`") nic={{ data_if }} op monitor interval=30s
```

Comment:
So why do you not change this `ClusterIP` resource creation? This command also accepts `cidr_netmask` parameter.